packages/tinacms/src/admin/pages/CollectionListPage.tsx

```diff
@@ -149,9 +149,13 @@ const CollectionListPage = () => {
                           <table className="table-auto shadow bg-white border-b border-gray-200 w-full max-w-full rounded-lg">
                             <tbody className="divide-y divide-gray-150">
                               {documents.map((document) => {
+                                const subfolders = document.node.sys.breadcrumbs
+                                  .slice(0, -1)
+                                  .join('/')
+
                                 return (
                                   <tr
-                                    key={`document-${document.node.sys.filename}`}
+                                    key={`document-${document.node.sys.relativePath}`}
```

Comment:
More unique in situations where you have two Documents of the same name, but in different folders.